### PR TITLE
Create CHANGELOG.md

### PR DESCRIPTION
**Pull request by ricardoas**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# 0.1.0
+#### Notes
+This is the first release of the OneView SDK in Python and it adds support to core features listed bellow.
+It is compatible with HPE OneView Rest API version 300 or higher.
+
+#### Redfish features supported
+ - Service Root
+ - System collection
+ - System
+ - System (Boot, Processor and Memory maapings)
```
Comment (llvieira) [inline CHANGELOG.md]: There is a typo with the word **mappings**.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# 0.1.0
+#### Notes
+This is the first release of the OneView SDK in Python and it adds support to core features listed bellow.
```
Comment (adriane-cardozo) [inline CHANGELOG.md]: first release of "OneView Redfish toolkit" in Python...

**Comment by bsoutham** [comment]
>> OneView Redfish Toolkit in Python 
I don't think we need to say in Python
just:  ... of the OneView Redfish Toolkit 

>>bellow --> below

**Comment by bsoutham** [comment]
How about using:

This is the first release of the OneView Redfish Toolkit.  It is compatible with HPE OneView REST API version 300 or higher.  Initial release supports the following features:

#### Redfish features supported
 - Service Root
 - System collection
 - System
 - System (Boot, Processor and Memory mappings)
 - Chassis collection
 - Chassis
 - Thermal mappings for Chassis
 - Manager collection
 - Manager
 - Error responses

**Pull request merged**